hive_agent/store/store.py

```diff
@@ -1,36 +1,41 @@
+import logging
+import sys
 from datetime import datetime
 
 from sqlalchemy import delete, update
 from sqlalchemy.future import select
 
 from hive_agent.store import DataEntry
 
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))
 
 class Store:
     def __init__(self, session_factory):
         self.session_factory = session_factory
+        logging.info("Database session initialized")
 
     async def add(self, namespace, data):
         async with self.session_factory() as session:
             new_entry = DataEntry(namespace=namespace, data=data, timestamp=datetime.now())
             session.add(new_entry)
             await session.commit()
             await session.refresh(new_entry)
-
+            logging.info(f"Added new entry: {new_entry.id} in namespace {namespace}")
```

Comment:
let's use `logging.debug` in this method and the ones after it since we don't want it to always show